.github/workflows/release-app.yml

```diff
@@ -110,11 +110,9 @@ jobs:
 
       - name: Add new changelog file to documentation
         run: |
-          cat << EOF > documentation/changelogs/${{ inputs.app }}/${{ steps.tag.outputs.date }}.md
-          # ${{ steps.tag.outputs.date }}
-
-          ${{ steps.release-drafter.outputs.body }}
-          EOF
+          file="documentation/changelogs/${{ inputs.app }}/${{ steps.tag.outputs.date }}.md"
+          echo -e "# ${{ steps.tag.outputs.date }}\n\n" > "$file"
+          echo -e '${{ steps.release-drafter.outputs.body }}' >> "$file"
```

Comment:
Was it bash here that removed the backticks? 🤯

Reply:
Yep! I mention this in the linked issue 😄 https://github.com/WordPress/openverse/issues/1189#issuecomment-1507613278